### PR TITLE
add expunge function to base store

### PR DESCRIPTION
**Pull request by netama**

### CODE REVIEW

File: microcosm_postgres/store.py [hunk 1]
```diff
@@ -51,6 +51,9 @@ def new_object_id(self):
         """
         return new_object_id()
 
+    def expunge(self, instance):
```
Comment (srt32) [inline microcosm_postgres/store.py]: Is expunge the complement to `add`?

**Comment by srt32** [comment]
LGTM

File: microcosm_postgres/store.py (hunk 1 above)
Comment (netama) [inline microcosm_postgres/store.py]: Yes (it detaches the object from the session).

**Comment by netama** [comment]
Thanks!

**Pull request merged**